client/src/partials/debtor_groups/debtor_groups.html

```diff
@@ -0,0 +1,281 @@
+<!-- breadcrumb header -->
+<bh-breadcrumb
+  path="DebtorGroupCtrl.bcPaths"
+  button="DebtorGroupCtrl.bcButtons">
+</bh-breadcrumb>
+
+<!-- main content -->
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-md-6">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            {{ "DEBTOR_GRP.GROUPS" | translate }}
+          </div>
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+                <th>#</th>
+                <th>{{ "COLUMNS.NAME" | translate }}</th>
+                <th>{{ "DEBTOR_GRP.IS_CONVENTION" | translate }}</th>
+                <th>{{ "DEBTOR_GRP.LOCKED" | translate }}</th>
+                <th>{{ "COLUMNS.ACTION" | translate }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="group in DebtorGroupCtrl.debtorGroupList | orderBy: 'name'">
+                <td>{{ $index + 1 }}</td>
+                <td>{{ group.name }}</td>
+                <td>
+                  <i ng-if="group.is_convention" class="glyphicon glyphicon-ok-sign"></i>
```

Comment:
In the future, for simply hiding small icons or pieces of HTML, `ng-show` is more efficient (save expensive adding and removing of DOM nodes).